python/gui/qgsmapcanvas.sip

```diff
@@ -276,6 +276,9 @@ class QgsMapCanvas : QGraphicsView
     //! true if antialising is enabled
     bool antiAliasingEnabled() const;
 
+    //! sets map tile rendering flag
+    void enableMapTileRendering( bool theFlag );
```

Comment:
I don't mind if 'set' or 'enable'. Just thought enableMapTileRendering would be consistent with enableAntiAliasing.